app/models/miq_template.rb

```diff
@@ -37,4 +37,8 @@ def self.without_volume_templates
   end
 
   def active?; false; end
+
+  def self.display_name(number = 1)
+    n_('VM Template and Image', 'VM Templates and Images', number)
```

Comment:
The wording seems inconsistent.  In another case it was "First thing / Second thing".  Also, Should it be "Templates and Images"?